app/src/main/java/com/asma/tasky/feature_management/presentation/components/ReminderSelector.kt

```diff
@@ -0,0 +1,63 @@
+package com.asma.tasky.feature_management.presentation.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.KeyboardArrowRight
+import androidx.compose.material.icons.outlined.Notifications
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.asma.tasky.core.presentation.ui.theme.LightGray
+import com.asma.tasky.core.presentation.ui.theme.SpaceMedium
+
+@Composable
+fun ReminderSelector() {
+    Row(
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .padding(horizontal = SpaceMedium)
+            .fillMaxWidth()
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Icon(
+                imageVector = Icons.Outlined.Notifications,
+                contentDescription = "reminder icon",
+                modifier = Modifier
+                    .background(
+                        color = LightGray,
+                        shape = RoundedCornerShape(2.dp)
+                    )
+                    .padding(2.dp),
+                tint = Color.DarkGray
+            )
+            Text(
+                text = "30 minutes before",
+                modifier = Modifier.padding(start = SpaceMedium),
+                fontWeight = FontWeight.Normal,
+                fontSize = 18.sp
+            )
+        }
+
+        IconButton(onClick = { /*TODO*/ }) {
+            Icon(
+                imageVector = Icons.Outlined.KeyboardArrowRight,
+                contentDescription = "reminder icon"
+            )
+        }
+    }
+}
```

Comment:
This looks unfinished, but I'd recommend to create some kind of sealed or enum class for the different reminder types which then map to a certain text (such as `30 minutes before`)